plugins/providers/policytag/client.go

```diff
@@ -0,0 +1,183 @@
+package policytag
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	datacatalog "cloud.google.com/go/datacatalog/apiv1"
+	"github.com/odpf/guardian/domain"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	pb "google.golang.org/genproto/googleapis/cloud/datacatalog/v1"
+	iampb "google.golang.org/genproto/googleapis/iam/v1"
+)
+
+type policyTagClient struct {
+	policyManager *datacatalog.PolicyTagManagerClient
+	projectId     string
+}
+
+func newPolicyTagClient(projectID string, credentialsJSON []byte) (*policyTagClient, error) {
+	ctx := context.Background()
+	policyManager, err := datacatalog.NewPolicyTagManagerClient(ctx, option.WithCredentialsJSON(credentialsJSON))
+	if err != nil {
+		return nil, err
+	}
+	return &policyTagClient{
+		policyManager: policyManager,
+		projectId:     projectID,
+	}, nil
+}
+
+func (p *policyTagClient) GetPolicies(ctx context.Context) ([]*Policy, error) {
+	taxonomies := p.policyManager.ListTaxonomies(ctx, &pb.ListTaxonomiesRequest{
+		Parent: p.toParent(p.projectId, US),
```

Comment:
Any particular reason, why location is hardcoded as `US` ?

Reply:
take resource name as projects/{project_id}/locations/{location}